projects/1.16/assets/aiot-botania/aiotbotania/lang/zh_cn.json

```
@@ -1,51 +1,68 @@
-  "botania.page.livingrockTools1": "$(1)活石剑$(0)",
-  "botania.page.livingrockTools0": "植物魔法全能工具模组可以将$(1)活石$(0)制成工具。<br>这些工具有与$(1)石质$(0)类似的特性，却在可操作性，耐用性和一些调整方面有些优越。每当$(1)活石$(0)物品受到损坏时，如果物品栏中有$(4)魔力$(0)（例如$(4)魔力石板$(0)）可用，它将代替它。<br>除此之外，它们与原版工具一样。",
-  "botania.page.livingwoodTools7": "$(1)活木全能工具$(0)",
-  "botania.page.livingwoodTools6": "$(1)活木锄$(0)",
-  "botania.entry.livingrockTools": "活石工具",
-  "botania.entry.livingwoodTools": "活木工具",
-  "item.aiotbotania.manasteel_aiot": "魔钢全能工具",
-  "item.aiotbotania.livingrock_aiot": "活石全能工具",
-  "item.aiotbotania.livingrock_hoe": "活石锄",
-  "item.aiotbotania.livingrock_shovel": "活石锹",
-  "item.aiotbotania.livingrock_pickaxe": "活石镐",
-  "item.aiotbotania.livingrock_axe": "活石斧",
-  "item.aiotbotania.livingrock_sword": "活石剑",
-  "item.aiotbotania.livingrock_shears": "活石剪刀",
-  "item.aiotbotania.livingwood_aiot": "活木全能工具",
-  "item.aiotbotania.livingwood_hoe": "活木锄",
-  "item.aiotbotania.livingwood_shovel": "活木锹",
-  "item.aiotbotania.livingwood_pickaxe": "活木镐",
-  "item.aiotbotania.livingwood_axe": "活木斧",
-  "item.aiotbotania.livingwood_sword": "活木剑",
-  "item.aiotbotania.livingwood_shears": "活木剪刀",
-  "aiotbotania.utilityMode": "§e放置火把和树苗",
-  "aiotbotania.hoeModePath": "§3耕作泥土和创建草径",
-  "aiotbotania.hoeMode": "§3耕作泥土",
-  "aiotbotania.toggleMode": "§6模式：",
-  "itemGroup.aiotbotania": "植物魔法全能工具"
+{
+  
+  "itemGroup.aiotbotania": "植魔全能工具",
+  "aiotbotania.toggleMode": "\u00a76模式:",
+  "aiotbotania.hoeModePath": "\u00a7a开垦土地与草径",
+  "aiotbotania.utilityMode": "\u00a7e放置火把与树苗",
+  "aiotbotania.mythicbotany.disabled": "请安装 MythicBotany！",
+  "aiotbotania.hoeMode": "\u00A73锄地",
+
+  "item.aiotbotania.livingwood_shears": "活木剪刀",
+  "item.aiotbotania.livingwood_sword": "活木剑",
+  "item.aiotbotania.livingwood_axe": "活木斧",
+  "item.aiotbotania.livingwood_pickaxe": "活木镐",
+  "item.aiotbotania.livingwood_shovel": "活木铲",
+  "item.aiotbotania.livingwood_hoe": "活木锄",
+  "item.aiotbotania.livingwood_aiot": "活木全能工具",
+
+  "item.aiotbotania.livingrock_shears": "活石剪刀",
+  "item.aiotbotania.livingrock_sword": "活石剑",
+  "item.aiotbotania.livingrock_axe": "活石斧",
+  "item.aiotbotania.livingrock_pickaxe": "活石镐",
+  "item.aiotbotania.livingrock_shovel": "活石铲",
+  "item.aiotbotania.livingrock_hoe": "活石锄",
+  "item.aiotbotania.livingrock_aiot": "活石全能工具",
+
+  "item.aiotbotania.manasteel_aiot": "魔钢全能工具",
+
+  "item.aiotbotania.elementium_aiot": "源质钢全能工具",
+  "item.aiotbotania.terra_shovel": "泰拉铲",
+  "item.aiotbotania.terra_hoe": "泰拉锄",
+  "item.aiotbotania.terra_aiot": "泰拉全能工具",
+
+  "item.aiotbotania.alfsteel_shovel": "精灵钢铲",
+  "item.aiotbotania.alfsteel_hoe": "精灵钢锄",
+  "item.aiotbotania.alfsteel_aiot": "精灵钢全能工具",
+
+  "botania.entry.livingwoodTools": "活木工具",
+  "botania.entry.livingrockTools": "活石工具",
+
+  "botania.page.livingwoodTools0": "若安装植魔全能工具模组，$(1)活木$(0)可以被制造成工具。<br>这些工具有着$(1)木头工具$(0)，不过在附魔和耐久上有较大优势。$(1)活木$(0)工具将会消耗4魔力来破坏方块，如果在背包中有$(4)储魔物品$(0)(就比如说一块$(1)魔力石板$(0))，就会优先消耗石板中的魔力。<br>除了伤害以外，他们和原版工具没什么两样。",

```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
  "botania.page.livingwoodTools0": "若安装植魔全能工具模组，$(1)活木$(0)可以被制造成工具。<br>这些工具的属性与$(1)木工具$(0)类似，不过在附魔和耐久上有较大优势。$(1)活木$(0)工具将会消耗4魔力来破坏方块，如果在背包中有$(4)储魔物品$(0)(就比如说一块$(1)魔力石板$(0))，就会优先消耗石板中的魔力。<br>除了伤害以外，他们和原版工具没什么两样。",
```
其余地方有类似的错误。

---

木制工具会更好吧

---

也可以，我基本上是混用。

---

已修正